pyccl/baryons/baccoemu_baryons.py

```diff
@@ -0,0 +1,169 @@
+__all__ = ("BaccoemuBaryons",)
+
+import numpy as np
+
+from .. import Pk2D
+from . import Baryons
+
+
+class BaccoemuBaryons(Baryons):
+    """ The baryonic boost factor computed with the baccoemu baryons emulators.
+
+    See https://arxiv.org/abs/2011.15018 and
+    https://bacco.dipc.org/emulator.html
+
+    Args:
+        log10_M_c (:obj:`float`): characteristic halo mass to model baryon
+                                   mass fraction
+        log10_eta (:obj:`float`): extent of ejected gas
+        log10_beta (:obj:`float`): slope of power law describing baryon mass
+                                    fraction
+        log10_M1_z0_cen (:obj:`float`): characteristic halo mass scale for
+                                         central galaxies
+        log10_theta_out (:obj:`float`):  outer slope of density profiles of
+                                          hot gas in haloes
+        log10_theta_inn (:obj:`float`): inner slope of density profiles of
+                                         hot gas in haloes
+        log10_M_inn (:obj:`float`): transition mass of density profiles of
+                                     hot gas in haloes
+    """
+    name = 'BaccoemuBaryons'
+    __repr_attrs__ = __eq_attrs__ = ("log10_M_c", "log10_eta", "log10_beta",
+                                     "log10_M1_z0_cen", "log10_theta_out",
+                                     "log10_theta_inn", "log10_M_inn")
+
+    def __init__(self, log10_M_c=14, log10_eta=-0.3, log10_beta=-0.22,
+                 log10_M1_z0_cen=10.5, log10_theta_out=0.25,
+                 log10_theta_inn=-0.86, log10_M_inn=13.4):
+        # avoid tensorflow warnings
+        import warnings
+        with warnings.catch_warnings():
+            warnings.filterwarnings('ignore', category=UserWarning)
+            import baccoemu
+            self.mpk = baccoemu.Matter_powerspectrum()
+        self.a_min = self.mpk.emulator['baryon']['bounds'][-1][0]
+        self.a_max = self.mpk.emulator['baryon']['bounds'][-1][1]
+        self.k_min = self.mpk.emulator['baryon']['k'][0]
+        self.k_max = self.mpk.emulator['baryon']['k'][-1]
+
+        self.bcm_params = {
+            'M_c': log10_M_c,
+            'eta': log10_eta,
+            'beta': log10_beta,
+            'M1_z0_cen': log10_M1_z0_cen,
+            'theta_out': log10_theta_out,
+            'theta_inn': log10_theta_inn,
+            'M_inn': log10_M_inn
+        }
+
+    def _sigma8tot_2_sigma8cold(self, emupars, sigma8tot):
```

Comment:
Can you add a test for the case where you pass A_s to the baryons emulator? Sorry I missed this on the first pass.

Reply:
done